src/tasks/common-tasks.ts

```
@@ -139,7 +139,7 @@ export namespace CommonTasks {
     return {
       title: '',
       enabled: () => false,
-      task: async () => {},
+      task: async () => { },
```

ESLint

[error] Do not add spaces between braces.

(unicorn/empty-brace-spaces)